src/legacy/core_plugins/kibana/public/visualize/editor/editor.js

```diff
@@ -232,7 +232,13 @@ function VisEditor(
       return !vis.hasInspector || !vis.hasInspector();
     },
     run() {
-      vis.openInspector().bindToAngularScope($scope);
+      const inspectorSession = vis.openInspector();
```

Comment:
We only used the `bindToAngularScope` implementation here, and since we don't want to have any Angular code in new platform, I moved the implementation here.